src/test/mir-opt/instrument_coverage.bar.InstrumentCoverage.diff

```diff
@@ -3,74 +3,16 @@
-+                                          // ty::Const
-+                                          // + ty: u32
-+                                          // + val: Value(Scalar(0x00000013))
-+                                          // mir::Constant
-+                                          // + span: /the/src/instrument_coverage.rs:19:18: 19:18
-+                                          // + literal: Const { ty: u32, val: Value(Scalar(0x00000013)) }
-+                                          // ty::Const
-+                                          // + ty: u32
-+                                          // + val: Value(Scalar(0x00000012))
-+                                          // mir::Constant
-+                                          // + span: /the/src/instrument_coverage.rs:19:18: 19:18
-+                                          // + literal: Const { ty: u32, val: Value(Scalar(0x00000012)) }
-+                                          // ty::Const
-+                                          // + ty: u32
-+                                          // + val: Value(Scalar(0x00000015))
-+                                          // mir::Constant
-+                                          // + span: /the/src/instrument_coverage.rs:19:18: 19:18
-+                                          // + literal: Const { ty: u32, val: Value(Scalar(0x00000015)) }
-+                                          // ty::Const
-+                                          // + ty: u32
-+                                          // + val: Value(Scalar(0x00000002))
-+                                          // mir::Constant
-+                                          // + span: /the/src/instrument_coverage.rs:19:18: 19:18
-+                                          // + literal: Const { ty: u32, val: Value(Scalar(0x00000002)) }
-+     }
-+ 
-+     bb1 (cleanup): {
-+         resume;                          // scope 0 at /the/src/instrument_coverage.rs:19:1: 21:2
-+     }
-+ 
-+     bb2: {
-+         StorageDead(_1);                 // scope 0 at /the/src/instrument_coverage.rs:20:5: 20:9
           _0 = const true;                 // scope 0 at /the/src/instrument_coverage.rs:20:5: 20:9
                                            // ty::Const
                                            // + ty: bool
                                            // + val: Value(Scalar(0x01))
                                            // mir::Constant
                                            // + span: /the/src/instrument_coverage.rs:20:5: 20:9
                                            // + literal: Const { ty: bool, val: Value(Scalar(0x01)) }
++         Coverage { kind: Counter { function_source_hash: 10208505205182607101, id: CounterValueReference(0) }, region: CodeRegion { file_name: "/the/src/instrument_coverage.rs", start_line: 19, start_col: 18, end_line: 21, end_col: 2 } }; // scope 0 at /the/src/instrument_coverage.rs:21:2: 21:2
```

Comment:
It would be nice to make this a little prettier. Using the standard line:column notation like `19:18 - 21:2` would help a bit.

Reply:
done